charts/dawarich/values.yaml

```diff
@@ -104,6 +105,52 @@ photon:
 #    limits:
 #      memory: 512Mi
 
+  loadbalancerProxy:
+    image:
+      registry: docker.io
+      repository: envoyproxy/envoy
+      tag: v1.31-latest
+      pullPolicy: IfNotPresent
+    podSecurityContext: {}
+    photonHosts:
+      - host: photon.komoot.io
+        scheme: https
+        port: 443
```

Comment:
Scheme and port should default to https and 443, they only need to be set when they differ.